src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -442,13 +438,15 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
       // In theory it shouldn't enter here, but we show a nice error page anyway.
 
       return (
-        <EmptyState className={emptyStateStyle}>
-          <EmptyStateIcon name="error-circle-o" />
-          <EmptyStateTitle>Error loading object `{this.props.match.params.object}`</EmptyStateTitle>
-          <EmptyStateInfo>
+        <EmptyState variant={EmptyStateVariant.full}>
+          <EmptyStateIcon color={PFAlertColor.Danger} icon={KialiIcon.Error} />
+          <Title headingLevel="h5" size="lg">
```

Comment:
nitpick: should level and size would be a constants instead? TitleLevel.h5 and TitleSize.lg?

Reply:
We should move this to a general file